### ISSUE TITLE
"Invalid Date" appears in patch review UI

### ISSUE THREAD

**Issue by rybesh** [periodo/periodo-client#97]
Haven't had a chance to dig into where this is happening; it might be a server issue rather than a client issue:
<img width="390" alt="screen shot 2015-09-18 at 1 08 51 pm" src="https://cloud.githubusercontent.com/assets/84592/9958107/e385452e-5e06-11e5-8aa9-2064fac0f126.png">

**Comment by rybesh** [comment]
Note that the submission date for the same patch looks OK here:
<img width="669" alt="screen shot 2015-09-18 at 1 14 02 pm" src="https://cloud.githubusercontent.com/assets/84592/9958132/342c7902-5e07-11e5-9805-493a60cfc3f5.png">

**Comment by ptgolden** [comment]
I hope this happened in a demo

**Comment by rybesh** [comment]
Of course it did!

**Comment by ptgolden** [comment]
Using the string as formatted by the server:

```javascript
console.log(new Date('2015-10-29T02:32:10+00.00'));
// > Invalid Date
```

I think this is a bug in the server. The timezone portion of the stamp should be `00:00`, not `00.00`.

**Comment by ptgolden** [comment]
it's already fixed (periodo/periodo-server@418df77c3706a8b7969b805fc6ae22407d4cdbc3), we just haven't deployed in a bit

**Comment by ptgolden** [comment]
Nevermind, it was a bug in the server. Fixed in https://github.com/periodo/periodo-server/commit/19ff928a1dbe7654776d0d0441f1509e909b196a
